modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -416,6 +417,14 @@ impl ServerModule for Wallet {
         &'a self,
         dbtx: &mut DatabaseTransaction<'_>,
     ) -> Vec<WalletConsensusItem> {
+        if tokio::time::timeout(Duration::from_secs(5), self.propose_citem.notified())
```

Comment:
Is there anything special about the desired interval of 1 sec in `submit_module_ci_proposals`? If so, it might not be ideal to set the timeout here longer

https://github.com/fedimint/fedimint/blob/6b8953fd79c8b30e60cacfff65287f3a5f46be2f/fedimint-server/src/consensus/mod.rs#L255